src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1168,6 +1168,31 @@
          (whitelist-matches? autosign subject))
        false))))
 
+(schema/defn ensure-cn-as-san :- utils/SSLExtension
+  "Given an SSLExtension for alt names and a cn, ensure that the cn is listed as a DNS alt name."
```

Comment:
However, I might write out "common name" here in the comment, or at least capitalize the acronym.

Reply:
see, I didn't even know it was "common name". I was gonna guess "certificate name". 😝